crates/edr_provider/src/data.rs

```diff
@@ -297,6 +297,31 @@ impl ProviderData {
         &self.instance_id
     }
 
+    pub fn interval_mine(&mut self) -> Result<bool, ProviderError> {
+        let result = self.mine_and_commit_block(None)?;
+
+        let header = result.block.header();
+        let is_empty = result.block.transactions().is_empty();
+        if is_empty {
+            self.logger.print_interval_mined_block_number(
+                header.number,
+                is_empty,
+                header.base_fee_per_gas,
+            );
+        } else {
+            log::error!("TODO: interval_mine: log mined block");
```

Comment:
I'm wondering, did you want to handle this as part of this PR?

Reply:
No, I don't really want to handle implement all of the logging logic here. I'm merely adding the `log::error!` to prevent someone from having to backtrack and reverse engineer which parts of the method haven't been implemented yet.

The control flow is the same as in HH. The `logMinedBlock` method is just missing.